CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * **(breaking change)** Removed `ifdef` from the language
 * **(breaking change)** Removed `PointerIO`
 * **(breaking change)** The `body` property of `HTTP::Request` is now an `IO?` (previously it was `String`). Use `request.body.try(&.gets_to_end)`  if you need the entire body as a String.
+* **(breaking change)** `MemoryIO` has been renamed to `IO::Memory`. The old name can still be used but will produce a compile-time warning. `MemoryIO` will be removed immediately after 0.20.0.
```

Comment:
Yes, that's what I meant. MemoryIO is gone but it's still there to ease migration. I want to remove it as soon as possible. Or we can do it in 0.21.0. In any case we are at 0.x so semver doesn't truly applies.